### DISCUSSION TITLE
How do you make a HTML file on Google Sites?

### DISCUSSION THREAD

**Discussion by alexnm27** [dragon731012/-WORKING-bookmarklets-and-games#5] [Q&A] [answered]
like the among us one

**Comment by dragon731012** [comment, marked as answer]
That is something I found on GitHub. It is very difficult, but if you really want to make one, you go to a page on Google sites, right click, click embed, then click code and write down the code. It is really difficult, and usually people use another website that isn’t blocked yet and do a url embed which is easier but if the site gets blocked, it does too, at least if your blocking system is securly or something like it. I hope that answers your question!
